### ISSUE TITLE
Troubleshooter checks only x86 homebrew location

### ISSUE THREAD

**Issue by SMillerDev** [gavinbenda/platinum-md#135]
Seeing the discussion https://github.com/Homebrew/discussions/discussions/664#discussioncomment-1699942, this app only checks `/usr/local/bin/brew` which means users ca very well have brew installed but the troubleshooting will not pick it up.

Running `which brew` would be a much better option or alternatively at least checking `/opt/homebrew/bin/brew`

**Comment by carlocab** [comment]
It looks like they do use `which brew`:

https://github.com/gavinbenda/platinum-md/blob/e12bd4fce080909edc34fbe9c4b8d18efe457902/src/renderer/components/LandingPage.vue#L424-L425

The issue is that `/opt/homebrew/bin` is not in `PATH` when that's done.

**Comment by DaveFlashNL** [comment]
but how to fix?

**Comment by carlocab** [comment]
> but how to fix?

https://github.com/Homebrew/discussions/discussions/664#discussioncomment-1700167

**Comment by DaveFlashNL** [comment]
https://github.com/Homebrew/discussions/discussions/664#discussioncomment-1701022

**Comment by deenine** [comment]
Can you view the hind device in finder?

**Comment by DaveFlashNL** [comment]
> Can you view the himd device in finder?

yes, it shows up in finder just fine, also in windows 11 vm too, which is were I've been using the terrible sonic stage Sony application to put music on the blanks
